docs/upgrade_to_tails_2x.rst

```diff
@@ -147,4 +147,4 @@ Next, repeat step 1 to restore a USB stick to a pristine state. While it probabl
 .. |Unwanted Bloatware Partition| image:: images/backup_and_migrate/partition_create_7.png
 .. |Root Terminal| image:: images/backup_and_migrate/root_terminal_3.png
 .. |Navigate to TailsData_unlocked| image:: images/backup_and_migrate/tails_data_unlocked_2.png
-.. |Upgrade by cloning| image:: images/backup_and_migrate/tails_installer_2.png
\ No newline at end of file
+.. |Upgrade by cloning| image:: images/backup_and_migrate/tails_installer_2.png
```

Comment:
This rather confusing diff means that a trailing newline was added when I saved the file. It's essentially a no-op.